pkg/relay/ethereum/tx.go

```diff
@@ -56,9 +59,11 @@ func (c *Chain) SendMsgs(msgs []sdk.Msg) ([]core.MsgID, error) {
 		// case *transfertypes.MsgTransfer:
 		// 	err = c.client.transfer(msg)
 		default:
+			logger.Error("failed to send msg", errors.New("illegal msg type"), "msg", msg)
 			panic("illegal msg type")
 		}
 		if err != nil {
+			logger.Error("failed to send msg", err, "msg", msg)
 			return nil, err
 		}
 		if receipt, revertReason, err := c.client.WaitForReceiptAndGet(ctx, tx.Hash(), c.config.EnableDebugTrace); err != nil {
```

Comment:
@dongrie please add logging to these error cases.

Reply:
Added `logger.Error`
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/13/commits/4611488cb01d764157417b2ccbf882ec9e47d78f

Reply:
@dongrie also add logging to the next else-if clause.

Reply:
Sorry, I didn't check
I added `logger.Error` here as well
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/13/commits/2f9a74a137bf324cfb11b70b760d9b0313b96a4e